runtime/onert/core/src/backend/controlflow/UserTensor.cc

```diff
@@ -35,6 +38,16 @@ size_t UserTensor::calcOffset(const ir::Coordinates &coords) const
   return offset;
 }
 
+bool UserTensor::applyShape(const ir::Shape &new_shape)
+{
+  // User tensors cannot be reallocated.
```

Comment:
I wonder whether this is true.
Is there any case where input size increases while calling `nnfw_run()` multiple times?

Reply:
This is the description for `UserTensor`.

```cpp
/**
 * @brief Tensor object that is for Input and Output tensors from the user.
 *
 * This class is a wrapped buffer that is allocated by the user. So it does not have resposibility
 * on allocation nor deallocation. All the model input/output tensors are wrapped with this class
 * for execution.
 *
 */
```

This tensor is just a wrapper of a buffer given by `nnfw_set_input/output`, so the buffer can never be changed during execution.

Reply:
Are you talking about `nnfw_run` could make some changes the input tensors? We don't have such models yet and not sure those models are useful. Even if those cases exist, my answer would still be the same.

Reply:
I see. Thanks.